src/Symfony/Component/Form/Form.php

```diff
@@ -748,7 +748,7 @@ public function isEmpty()
 
         return FormUtil::isEmpty($this->modelData) ||
             // arrays, countables
-            0 === count($this->modelData) ||
+            ((is_array($this->modelData) || $this->modelData instanceof \Countable) && 0 === count($this->modelData)) ||
```

Comment:
This is to avoid calling `count()` on an array? If that's the case, shouldn't we fix this throughout all `0 === count()` calls in the code base?